coderd/database/migrations/000306_update_protect_deleting_organization_function.down.sql

```diff
@@ -0,0 +1,77 @@
+-- Drop trigger that uses this function
+DROP TRIGGER IF EXISTS protect_deleting_organizations ON organizations;
+
+-- Revert the function to its original implementation
+CREATE OR REPLACE FUNCTION protect_deleting_organizations()
+	RETURNS TRIGGER AS
+$$
+DECLARE
+    workspace_count int;
+	template_count int;
```

Comment:
classic tabs vs spaces moment (you've got 4 spaces on one line and a tab on the next)